File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    var answer = '';
    for(let i = 1; i <= food.length; i++){
        for(let j = 0; j < Math.floor(food[i]/2); j++){
            answer += i.toString();
        }
    }
    answer+='0';
    for(let i = food.length; i >= 0; i--){
        for(let j = 0; j < Math.floor(food[i]/2); j++){
            answer += i.toString();
        }
    }
    return answer;
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.py
import math

def solution(food):
    answer = ''
    for i in range(1, len(food)):
        temp = food[i]
        if temp % 2 != 0:
            temp -= 1
        for j in range(temp // 2):
            answer += str(i)

    reverseAnswer = answer[::-1]
    return answer + '0' + reverseAnswer
